src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -227,9 +227,9 @@ export class EstimatedExecutionPlanKeyboardAction extends Action {
 	}
 }
 
-export class RunCurrentQueryWithActualPlanKeyboardAction extends Action {
-	public static ID = 'runCurrentQueryWithActualPlanKeyboardAction';
-	public static LABEL = nls.localize('runCurrentQueryWithActualPlanKeyboardAction', "Run Current Query with Actual Plan");
+export class ToggleActualPlanKeyboardAction extends Action {
+	public static ID = 'ToggleActualPlanKeyboardAction';
+	public static LABEL = nls.localize('ToggleActualPlanKeyboardAction', "Execution Plan: Enable/Disable Actual Execution Plan");
```

Comment:
You're merging in the changes from the other PR (adding the category) here.

And for that I believe you should be using the category parameter of registerWorkbenchAction, not embedding it in the label directly.